UPGRADING.md

```diff
@@ -1,5 +1,101 @@
 # Upgrade guidelines
 
+## 0.119.0 to 0.120.0
+
+This guide provides steps for new users, transitioning users, and those maintaining previously deployed Operator-related TLS certificates and configurations.
+
+- New users: No migration is required for Operator TLS certificates.
+- Previous users: Migration may be needed if using `operator.enabled=true` or `certmanager.enabled=true`.
+
+To maintain previous functionality and avoid breaking changes, review the following sections.
+
+### **Maintaining Previous Functionality via Helm Values Update**
+
+#### **Scenario 1: Operator and cert-manager Deployed via This Helm Chart**
+
+If you previously deployed both the Operator and cert-manager via this Helm chart (`operator.enabled=true` and `certmanager.enabled=true`), you can preserve functionality by adding the following values:
+
+```yaml
+operator:
+  enabled: true
+  admissionWebhooks:
```

Comment:
Does a user not need to explicitly disable the now true option `autoGenerateCert.enabled`?

Reply:
The user does not. Whenever `operator.admissionWebhooks.certManager.enabled=true` the usage of `operator.admissionWebhooks.autoGenerateCert.enabled` is nullified. Adding a doc update relating to this for a little more clean up.